src/main/java/Duke.java

```diff
@@ -6,5 +9,36 @@ public static void main(String[] args) {
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+
+        System.out.println("Hello! I'm Duke\n");
+        System.out.println(" What can I do for you?\n");
+
+        String array[] = new String[100];
```

Comment:
```suggestion
        String[] array = new String[100];
```
Your array declaration is C-style and not Java